### ISSUE TITLE
Code v3

### ISSUE THREAD

**Issue by ildemsanli** [ildemsanli/Mastermind#5]
Olivier, the changes you have made don't account for the positional information of the pins. We need to show which position is an exact or near match as we did before.

**Comment by aiolii63** [comment]
Yeah but i checked the rules of Mastermind and it seems like the game works this way !
Not showing where the exact or near matches are is a way to make the game more challenging...

**Comment by ildemsanli** [comment]
I don't think anybody would want that much of challenge for a kid's game 😄
